### PR TITLE
Ros restructuring

### PR DESCRIPTION
**Pull request by rbudai98**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ sdk/include/aditof/version.h
 
 *deps/
 *.temp/
+.vscode/
```
Comment (dNechita) [inline .gitignore:7]: new line is missing

File: bindings/ros/CMakeLists.txt [hunk 2]
```diff
@@ -21,5 +21,6 @@ if(UNIX)
     TARGET aditof_ros_package
     COMMAND ${CMAKE_BINARY_DIR}/ros_setup.bash "${OLD_CMAKE_INSTALL_PREFIX}"
             "${OLD_CMAKE_PREFIX_PATH}" ${VERSION} DEPENDS aditof)
- 
+
```
Comment (dNechita) [inline bindings/ros/CMakeLists.txt:24]: space added here looks to be added by mistake

File: bindings/ros/README.md [hunk 3]
```diff
@@ -2,69 +2,105 @@
...
-  sudo cp -r <config folder directory> $HOME/.ros/
+  sudo apt install ros-ROSDISTRO-PACKAGE
   ```
-  where "libs folder direcory" is the location of the folder where the depth_compute files are stored and the "config folder directory" is the directory where the neccessary config files are strored for the camera. \
-
-
-  1.4 After this, build the SDK, as well as enable ROS package building 
-
+- **Install the ADI ToF SDK library**
+  - [Install SDK dependencies](https://github.com/analogdevicesinc/aditof_sdk/blob/6c7fb376aeec73a21ab177adf297c5781bcbd544/doc/linux/build_instructions.md#installing-the-dependencies)
```
Comment (dNechita) [inline bindings/ros/README.md:18]: I would use a link that contains master and not a specific commit SHA like this one: 6c7fb376aeec73a21ab177adf297c5781bcbd544

File: .gitignore (hunk 1 above)
Comment (rbudai98) [reply .gitignore]: added

File: bindings/ros/CMakeLists.txt (hunk 2 above)
Comment (rbudai98) [reply bindings/ros/CMakeLists.txt:31]: removed

File: bindings/ros/README.md (hunk 3 above)
Comment (rbudai98) [reply bindings/ros/README.md]: Changed to new reference

File: sdk/CMakeLists.txt [hunk 4]
```diff
@@ -149,13 +149,13 @@ endif()
 if (USE_ITOF)
 	
 	if ( UNIX )
-    		add_library(libtofi_compute SHARED IMPORTED)
```
Comment (dNechita) [inline sdk/CMakeLists.txt:152]: Why static and not shared? I know you mention this some time ago but I forgot (sorry).

**Review by dNechita — APPROVED** [review]

**Comment by dNechita** [comment]
@rbudai98 is there anything else that you need to add to this PR?

**Pull request merged**